### PR TITLE
Tortle v1.0.0 - Initial Commit

### PR DESCRIPTION
**Pull request by thrifus**
Tiny utility for easily enabling or disabling Tor

Homepage:
https://thrifus.github.io/Tortle

### CODE REVIEW

File: Library/Formula/tortle.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+class Tortle < Formula
+  desc "Tortle is a tiny utility for easily enabling or disabling Tor"
+  homepage "https://thrifus.github.io/Tortle"
+  url "https://github.com/thrifus/Tortle/archive/1.0.2.tar.gz"
+  version "1.0.2"
+  sha256 "f9cb1de7586f99b9a1aa746a7e2b7f1939179841c479b243a3c8a36eff0c3186"
+
+  depends_on "tor"
```
Comment (bfontaine) [inline Library/Formula/tortle.rb]: Please leave an empty line below this one.

File: Library/Formula/tortle.rb [hunk 2]
```diff
@@ -0,0 +1,16 @@
+class Tortle < Formula
+  desc "Tortle is a tiny utility for easily enabling or disabling Tor"
```
Comment (bfontaine) [inline Library/Formula/tortle.rb]: You can shorten this as “Tiny utility for easily enabling or disabling Tor”.

File: Library/Formula/tortle.rb [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+  homepage "https://thrifus.github.io/Tortle"
+  url "https://github.com/thrifus/Tortle/archive/1.0.2.tar.gz"
+  version "1.0.2"
```
Comment (bfontaine) [inline Library/Formula/tortle.rb]: You can remove this line; Homebrew already gets the version from the URL.

**Comment by thrifus** [comment]
Thanks you for notifying me of what I should change, I'm fairly new to Ruby so I'm a tad lost

**Pull request closed without merging**